### PR TITLE
Limma update for edger/trinity

### PR DESCRIPTION
**Pull request by abretaud**
There is dependency problem with edger 3.12 that needs a limma version >= 3.25.6
This should fix the installation error encountered when installing Trinity.

### CODE REVIEW

File: tools/trinity/run_DE_analysis.xml [hunk 1]
```diff
@@ -7,52 +7,52 @@
         <requirement type="package" version="1.10.0">bioconductor-deseq2</requirement>
         <requirement type="package" version="3.12.0">edger</requirement>
```
Comment (bgruening) [inline tools/trinity/run_DE_analysis.xml]: Do we need to increase the tool version here?

Comment (yhoogstrate) [inline tools/trinity/run_DE_analysis.xml]: Yes

**Comment by nsoranzo** [comment]
Out of curiosity: edgeR package does not mention a minimum limma version, how did you find out this requirement?

File: tools/trinity/run_DE_analysis.xml (hunk 1 above)
Comment (abretaud) [inline tools/trinity/run_DE_analysis.xml]: Do you mean something like 3.12.0.1?

**Comment by abretaud** [comment]
I found it out while installing trinity, I got this error in the logs while edger dependency was being installed:
```
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:41,963 installing to /galaxy/dependencies/bioconductor-edger/3.12.0/iuc/package_r_edger_3_12_0/5c1431123155/edgeR/libs
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:41,963 ** R
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:42,064 ** inst
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:42,064 ** preparing package for lazy loading
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:43,367 Error : object ‘kegga’ is not exported by 'namespace:limma'
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:43,368 ERROR: lazy loading failed for package ‘edgeR’
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:43,368 * removing ‘/galaxy/dependencies/bioconductor-edger/3.12.0/iuc/package_r_edger_3_12_0/5c1431123155/edgeR’
tool_shed.galaxy_install.tool_dependencies.recipe.install_environment:bioconductor-edger DEBUG 2016-05-11 17:49:43,468 <simpleWarning in install.packages(c("edgeR_3.12.0_src_all.tar.gz"), lib = "/galaxy/dependencies/bioconductor-edger/3.12.0/iuc/package_r_edger_3_12_0/5c1431123155",     repos = NULL, dependencies = FALSE): installation of package ‘edgeR_3.12.0_src_all.tar.gz’ had non-zero exit status>
```
And looking at the limma code, I found that the kegga function was introduced in [3.25.6](https://github.com/Bioconductor-mirror/limma/commit/8bd3d3fb9aaa9ba8441389ff11d82550f0c3067d)

File: tools/trinity/run_DE_analysis.xml (hunk 1 above)
Comment (nsoranzo) [inline tools/trinity/run_DE_analysis.xml]: @abretaud: No, something like:
```xml
<tool id="run_DE_analysis" name="Differential expression analysis" version="2.1.1.1">
```

Comment (abretaud) [inline tools/trinity/run_DE_analysis.xml]: Oh, sure, it's done!

**Comment by nsoranzo** [comment]
:+1: Thanks @abretaud!

File: tools/trinity/tool_dependencies.xml [hunk 2]
```diff
@@ -21,8 +21,8 @@
     <package name="express" version="1.5.1">
         <repository name="package_express_1_5_1" owner="iuc" />
     </package>
-    <package name="limma" version="3.27.4">
-        <repository name="package_r_limma_3_27_4" owner="iuc" />
+    <package name="limma" version="3.28.2">
+        <repository name="package_r_limma_3_28.2" owner="iuc" />
```
Comment (nsoranzo) [inline tools/trinity/tool_dependencies.xml]: Typo fixed in commit 91515eb8b87ca2399f556369d6112b0cf3a78617.

**Comment by nsoranzo** [comment]
TTS and MTS updated.

**Comment by bgruening** [comment]
@nsoranzo thanks! We can now also bug @abretaud with updating the TSs - I send him the credentials a few minutes ago! :)

**Comment by nsoranzo** [comment]
Cool, welcome on board!

**Pull request merged**